### ISSUE TITLE
Bug with Flatpaks programs installed on multiple users and more.

### ISSUE THREAD

**Issue by stockbed** [pop-os/beta#165]
**How did you upgrade to 21.04? (Fresh install / Upgrade)**

_Fresh install_ 

 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**

**Issue/Bug Description:**

_1- I have two users in Pop!_OS.
When I install a program in Flatpak version (like Spotify or any other) the other user does not have the program installed.

The same thing happens with the audio and video codecs that come pre-installed, that is, the first user has them installed but the second one has to install them when trying to play something in Totem (Videos).

Both of these things happen since the previous version of Pop!_OS. It probably only happens to a few, as not many people use more than one user, still it's an annoying bug that if it's not fixed I'll be forced to look for another Ubuntu based distro. I love Pop!_OS but it's the only distro I've tried that has a bug like this.

2- For some strange reason the Bluetooth on/off switch has a bug. If I turn Bluetooth off I can't turn it back on, but have to reboot the laptop. 

HONORABLE MENTIONS: 
- Pop!_Shop freezes and crashes very often.
- Newly installed Pop!_OS 21.04 --> Open Pop!_Shop --> Surprise, a box in the middle of the screen saying: 
<< Could not get lock /var/lib/apt/lists/lock. It is held by process 5411 (apt)
N: Be aware that removing the lock file is not a solution and may break your system.
E: Unable to lock directory /var/lib/apt/lists/ >>

That error is because an APT process is running, I had to restart the laptop to finish the process and do a <<sudo apt update>> from the terminal.
In conclusion of this last error, Pop!_Shop is going very bad._

**Steps to reproduce (if you know):**

_For the Flatpaks error: Install any program in Flatpak (either from the Pop!_Shop or from the terminal using Flatpak commands), then create a new user and notice that the program is not installed on the second user._

**Expected behavior:**

Flatpaks appear installed on any user.

**Other Notes:**
None--.

**Comment by stockbed** [comment]
Thank you.

**Comment by stockbed** [comment]
For anyone who wants to fix this bug, add the Flathub report at the system level by running it from the terminal:

flatpak remote-add --system --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo

Restart and then remove the user-level repo from Pop!_Shop.

And that's it, any installed Flatpak will be installed for all users. 
Thanks.

**Comment by stockbed** [comment]
For anyone who wants to fix this bug (although it is not), add the Flathub report at the system level by running it from the terminal:

flatpak remote-add --system --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo

Restart and then remove the user-level repo from Pop!_Shop.

And that's it, any installed Flatpak will be installed for all users. 
Thanks.

**Comment by stockbed** [comment]
For anyone who wants to fix this bug (although it is not), add the Flathub report at the system level by running it from the terminal:

flatpak remote-add --system --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo

Restart and then remove the user-level repo from Pop!_Shop.

And that's it, any installed Flatpak will be installed for all users.
Thanks. @mmstick

**Comment by stockbed** [comment]
For anyone who wants to fix this bug (even if it is not), open the terminal, and:

    Delete the user-level Flathub repository with:

flatpak remote-delete flathub

    Logout and restart.

    Add the repository again, but this time at system level with:

flatpak remote-add --system --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo

    Logout and restart. Just in case you need to run:

sudo apt update

    Open the Pop!_Shop and you are done.
    Thanks. @mmstick

**Comment by stockbed** [comment]
For anyone who wants to fix this bug (even if it is not), open the terminal, and:

1. Delete the user-level Flathub repository with:

flatpak remote-delete flathub

2. Logout and restart. 

3. Add the repository again, but this time at system level with:

flatpak remote-add --system --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo

4. Logout and restart. Just in case you need to run:

sudo apt update

5. Open the Pop!_Shop and you are done.
Thx @mmstick
